docs(dev-server.md): Clarify the possible return value for devServer.… (#3211)
webpack/webpack.js.org: src/content/configuration/dev-server.md

Sometimes you don't want to proxy everything. It is possible to bypass the proxy based on the return value of a function.

In the function you get access to the request, response and proxy options.

> - Return `null` or `undefined` means continue to process the request with proxy.

Reviewer: ```suggestion
- Return `null` or `undefined` to continue processing the request with proxy.
```

Reviewer: Does it also mean, returning nothing, would do the same as `null` and `undefined`? Maybe worth mentioning

Author: Return nothing is essentially return `undefined`. I think it is clear enough.

> - Return `false` will produce a 404 error for the request.

Reviewer: ```suggestion
- Return `false` to produce a 404 error for the request.
```

> - If a string path is returned. The path will be served instead of continuing to proxy the request.

Reviewer: ```suggestion
- Return a path to serve from, instead of continuing to proxy the request.
```
